Gemfile

```diff
@@ -52,6 +52,7 @@ end
 group :lint do
   # Code style
   gem "rubocop", "0.91.0"
+  gem "rubocop-packaging", "~> 0.5"
```

Comment:
Not a big deal at all, since gems are locked in the `Gemfile.lock` file anyways, but the current criteria is to use `~> x.y` for gems that have reached 1.0, and an exact version for gem still in the 0.x phase. Could you use

```suggestion
  gem "rubocop-packaging", "0.5.0"
```

just to keep it consistent?

Reply:
Most certainly! I just committed the suggestion!